src/cls/IPM/Lifecycle/Module.cls

```diff
@@ -186,4 +186,48 @@ Method %Package(ByRef pParams) As %Status
   quit tSC
 }
 
+Method %Update(ByRef pParams) As %Status
+{
+    set tSC = $$$OK
+    try {
+        set tVerbose = $get(pParams("Verbose"))
+        set tVersion = $get(pParams("Version"))
+        set tModuleName = ..Module.DisplayName
+        set tOldModule = ##class(%IPM.Storage.Module).NameOpen(tModuleName)
+
+        write:tVerbose !,"Seeding update steps for original module version",!
+        $$$ThrowOnError(tOldModule.SeedUpdateSteps())
```

Comment:
Why are we ever seeding steps in Update? Seeding of steps would be done in Activate when a module is branch new